packages/grafana-data/src/types/panel.ts

```diff
@@ -80,6 +85,10 @@ export class PanelPlugin<TOptions = any> extends GrafanaPlugin<PanelPluginMeta>
   editor?: ComponentClass<PanelEditorProps<TOptions>>;
   customFieldConfigs?: FieldConfigEditorRegistry;
   defaults?: TOptions;
+  fieldConfigDefaults?: FieldConfigSource = {
+    defaults: {},
+    overrides: [],
+  };
```

Comment:
this feels a bit off, should it not be fieldConfigDefaults: FieldConfig 

Or maybe we want panels with default overrides I guess.  

thoguhts @ryantxu ?

Reply:
Good point, not really sure so I'll wait for Ryan to kick in.